python/tvm/relay/analysis/analysis.py

```diff
@@ -384,6 +384,21 @@ def extract_fused_functions(mod):
     return ret
 
 
+def extract_operators(mod):
```

Comment:
This name seems not accurate nor can reflect the functionality of this pass. Something like `list_used_op_names` or `list_appear_op_names` might be more precise.